packages/database/src/migrations/20210301123426-AddFetpDashboardsToSolomonIs-modifies-data.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+import { arrayToDbString, insertObject, updateValues } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const revertDashboardGroupCodes = ['SB_FETP_Country_FETP_Graduates', 'SB_FETP_Country_Public'];
+
+exports.up = async function (db) {
+  const dashboardGroups = (
+    await db.runSql(`
+    select * from "dashboardGroup" where 'fetp' = ANY("projectCodes") and "organisationUnitCode" IN ('PG', 'SB');
+  `)
+  ).rows;
+
+  const sbDashboardGroups = dashboardGroups
+    .filter(r => r.organisationUnitCode === 'SB')
+    .map(dg => dg.code);
+
+  const pgDashboardGroups = dashboardGroups.filter(r => r.organisationUnitCode === 'PG');
```

Comment:
Confusing that `pgDashboardGroups` contains the record, whereas `sbDashboardGroups` contains the codes. Would suggest aligning both to have the same data structure, or rename one

Reply:
Yep fair call, renamed to sbDashboardGroupCodes